Whiskey/Functions/Resolve-WhiskeyDotNetSdkVersion.ps1

```diff
@@ -100,22 +73,22 @@ function Resolve-WhiskeyDotNetSdkVersion
                 Select-Object -ExpandProperty 'version'
         }
 
-        $resolvedVersion =
+        $sortedSdkVersions =
             $sdkVersions |
-            ForEach-Object { $_ -as [Version] } |
-            Where-Object { $_ -like $Version } |
-            Sort-Object -Descending |
-            Select-Object -First 1

```

Comment:
We still need this because `$Version` accepts wildcards. Before we start checking versions against `$RollForward`, do this:

```powershell
if ([WildcardPattern]::ContainsWildcardCharacters($Version))
{
    return $sdkVersions |
           Where-Object { $_ -like $Version } |
           Sort-Object -Descending |
           Select-Object -First 1
}
```

Make sure `$sdkVersions` is already an array of `[Version]` objects.